app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -24,6 +24,7 @@ def create
         flash[:phone_error] = t('errors.two_factor_auth_setup.must_select_additional_option')
         redirect_to authentication_methods_setup_path(anchor: 'select_phone')
       else
+        flash[:error] = t('errors.two_factor_auth_setup.must_select_option')
```

Comment:
is this needed?

Reply:
Yes, right now if a user throws an error it just reloads the page with no indication to the user whats wrong, this will at least show a user an indication telling them to select the additional MFA method.  maybe we can just raise a Param error, but that wouldn't make sense.